### DISCUSSION TITLE
Frontend Initiatives Meeting - May 11, 2021

### DISCUSSION THREAD

**Discussion by varl** [dhis2/notes#283] [Meetings] [answered]
When: May 11, 2021 @ 15:00 CET
Where: Remote

# Agenda

-   General
    -   Plurals in i18n: https://github.com/dhis2/app-platform/pull/552
    -   Workflow updates for forks/Dependabot: https://github.com/dhis2/workflows
    -   cli-style version 8.x, beware of breaking changes: https://github.com/dhis2/cli-style/blob/master/CHANGELOG.md
    -   Client-side caching approach discussion: https://github.com/dhis2/notes/discussions/282

-   Working groups
    -   PWA
    -   Cypress
    -   CI/CD
    -   ui / DataTable progress
    -   PR previews

-  Other?

**Comment by edoardo** [comment]
`DataTable` release? 🙊

**Reply by varl** [reply]
Yes.

**Comment by varl** [comment, marked as answer]
PR for minutes: https://github.com/dhis2/notes/pull/284
